TestKickStart.bat

```diff
@@ -0,0 +1,8 @@
+
+@ECHO ON
+
+ECHO Python 2.7 Tests ===============================
+py -2.7 TestRunner.py
```

Comment:
I don't know if these count as virtual environments, just startup options. I tried using **virtualenv** but that didn't seem to like running in a BAT file, and I wasn't able to have it run my *TestRunner.py*. To me it feels like a time-crunched hack. But it works for the time being.